src/main/java/com/google/api/codegen/configgen/mergers/ProtoConfigMerger.java

```diff
@@ -57,14 +56,10 @@ private ConfigMerger createMerger(Model model, String fileName) {
   }
 
   private String getPackageName(Model model, ConfigHelper helper) {
-    if (model.getServiceConfig().getApisCount() > 0) {
-      Api api = model.getServiceConfig().getApis(0);
-      Interface apiInterface = model.getSymbolTable().lookupInterface(api.getName());
-      if (apiInterface != null) {
-        return apiInterface.getFile().getFullName();
-      }
+    String packageName = ProtoParser.getPackageName(model);
+    if (packageName != null) {
+      return packageName;
     }
-
     helper.error(model.getLocation(), "No interface found");
```

Comment:
Should we update this message to something like "Failed to determine package name"?

Reply:
Done.